src/web/utils/AppVisibilityUtils.ts

```diff
@@ -0,0 +1,96 @@
+/**
+* AppVisibilityUtils.ts
+*
```

Comment:
You can remove the ifvisible dependency in package.json.

Reply:
Yes i did that.